README.md

```diff
@@ -1,15 +1,15 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
+capstone1-concert-website
 
 > Description the project.
 
 
```

Comment:
- [ ] Kindly make sure your project has a description telling in detail what the project is about.